README.md

```diff
@@ -25,6 +25,9 @@ jobs:
     steps:
       # Drafts your next Release notes as Pull Requests are merged into "master"
       - uses: release-drafter/release-drafter@v5
+        with:
+          # (Optional) specify config name to use, relative to .github/. Default: release-drafter.yml
+          config-name: my-config.yml
```

Comment:
```suggestion
          # config-name: my-config.yml
```

Great way of explaining! I do want us to comment it out though as people are going to copy it and wonder why it doesn't work :)

Reply:
You're right! 😆